Sources/FluentKit/Properties/CompositeParent.swift

```diff
@@ -0,0 +1,163 @@
+extension Model {
+    public typealias CompositeParent<To> = CompositeParentProperty<Self, To>
+        where To: Model, To.IDValue: Fields
+}
+
+@propertyWrapper @dynamicMemberLookup
+public final class CompositeParentProperty<From, To>
```

Comment:
Hate to be that person but would be good to have some API docs for new public stuff